base/loading.jl

```diff
@@ -460,8 +466,13 @@ function _require(mod::Symbol)
                 eval(Main, :(Base.include_from_node1($path)))
 
                 # broadcast top-level import/using from node 1 (only)
-                refs = Any[ @spawnat p eval(Main, :(Base.include_from_node1($path))) for p in filter(x -> x != 1, procs()) ]
-                for r in refs; wait(r); end
+                @sync begin
+                    for p in filter(x -> x != 1, procs())
+                        @async remotecall_fetch(p) do
```

Comment:
It is better to use a `remotecall_fetch` (after ensuring that the remote execution returns `nothing`) when we we do not need the result of the remote computation. Will avoid unnecessary storage and and clean-up of the result on the remote node.